src/tests/base_test.py

```diff
@@ -821,6 +822,16 @@ def start_ipxe_server(**kwargs):
             for server in ipxe_server_controllers:
                 server.remove()
 
+    @pytest.fixture()
+    def iscsi_target_controller(self) -> Callable:
+        log.info("--- SETUP --- iscsi server controller")
+        iscsi_server_controller = IscsiTargetController()
```

Comment:
need to pass a receiver, no?

Reply:
Well  , we can do it but since its running on utils.run_command i adeed it inside in the singleton director. 
class IscsiTargetController:
  def __init__(self):
        self.builder = IscsiBuilder(Receiver(utils.run_command, shell=True))
        self._target_configs = []


Not sure if we want it from base_test ???  what do you think ?
Its more an option if the builder will run from VM in the future /

What do you think ?

Reply:
ack, let it like that for the moment